### PR TITLE
Little bit of styling

### PR DESCRIPTION
**Pull request by mikx98**

### CODE REVIEW

File: src/worklogs/components/WorkLogInput.js [hunk 1]
```diff
@@ -37,6 +37,8 @@ class WorkLogInput extends React.Component<Props> {
   handleKeyDown = e => {
     const { hints, historyIndex } = this.props
 
+    e.key === 'Enter' && this.handleSubmit(e)
+
     if (!(e.key in INCREMENT_BY_KEY)) return
```
Comment (Siegrift) [inline src/worklogs/components/WorkLogInput.js]: I have an eslint error here. I see that you turned this type of rule ("curly": ["error", "all"]) explicitely on in .eslintrc. Is there a reason for it? We should probably stich to vacuum config.

File: src/worklogs/components/WorkLogInput.js [hunk 2]
```diff
@@ -37,6 +37,8 @@ class WorkLogInput extends React.Component<Props> {
   handleKeyDown = e => {
     const { hints, historyIndex } = this.props
 
+    e.key === 'Enter' && this.handleSubmit(e)
```
Comment (Siegrift) [inline src/worklogs/components/WorkLogInput.js]:
```suggestion
    if (e.key === 'Enter') {
       this.handleSubmit(e)
       return
    }
```

File: src/worklogs/components/WorkLogInput.js (hunk 1 above)
Comment (mikx98) [reply src/worklogs/components/WorkLogInput.js]: it was a wish from @jakuub. As there are just a few people on this project, why not?

File: src/worklogs/components/WorkLogInput.js [hunk 3]
```diff
@@ -84,17 +89,15 @@ class WorkLogInput extends React.Component<Props> {
     const { value } = this.props
 
     return (
-      <form onSubmit={this.handleSubmit}>
```
Comment (Siegrift) [inline src/worklogs/components/WorkLogInput.js]: :+1:

File: src/worklogs/components/WorkLogInput.js (hunk 1 above)
Comment (Siegrift) [reply src/worklogs/components/WorkLogInput.js]: Just asking, I personally don't like it visually...

**Pull request merged**